content/en/blog/2023/2023-observability-predictions.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "2023 Observability Predictions: Open Source is the Key to Explosive Growth"
+linkTitle: 2023 Observability Predictions
+date: 2023-01-04
+author: "[Paul Stefanski](https://github.com/rpstefanski) ([observIQ](https://observiq.com/))"
+---
+
+## 2023 Observability Predictions: Open Source is the Key to Explosive Growth
```

Comment:
This heading isn't necessary since the blog layout get's the title from the `title` in the front matter.

Reply:
thanks for review! removed